python/lsst/pipe/tasks/calibrate.py

```diff
@@ -46,6 +46,16 @@
 from .computeExposureSummaryStats import ComputeExposureSummaryStatsTask
 
 
+class _EmptyTargetTask(pipeBase.PipelineTask):
```

Comment:
I would still add a doctoring to this to indicate it is only a placeholder, why, and that no one should use it (I know the _ is supposed to mean that, but more documentation is always better)